Lib/test/test_heapq.py

```diff
@@ -38,7 +38,7 @@ def load_tests(loader, tests, ignore):
     class HeapqMergeDocTestFinder:
         def find(self, *args, **kwargs):
             dtf = doctest.DocTestFinder()
-            return dtf.find(py_heapq.merge)
+            return dtf.find(py_heapq.merge, module=py_heapq)
```

Comment:
Complicated, but the thing is `py_heapq` isn't passed to the doctest finder because it's lazy at the time, so one has to be explicit about the module we're using. I can dig again why this was needed, but it made total sense once I figured out (now I forgot).